### ISSUE TITLE
Future incompatibility

### ISSUE THREAD

**Issue by puzzlewolf** [emgre/win-crypto-ng#40]
Running Rust 1.62.1's cargo check reports a future incompatibility, with the following report:
```
The package `win-crypto-ng v0.4.0` currently triggers the following future incompatibility lints:
> warning: reference to packed field is unaligned
>   --> /home/nora/.cargo/registry/src/github.com-1ecc6299db9ec823/win-crypto-ng-0.4.0/src/helpers/blob.rs:38:18
>    |
> 38 |         unsafe { &self.0 }
>    |                  ^^^^^^^
>    |
>    = note: `#[allow(unaligned_references)]` on by default
>    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
>    = note: for more information, see issue #82523 <https://github.com/rust-lang/rust/issues/82523>
>    = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
>    = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
> 
```
I don't know when this will become a problem, but I think you'd like to know :)

**Comment by emgre** [comment]
Thanks for letting me know, I'll investigate.

**Comment by ribeaud** [comment]
I think, we now just hit the next level: I am having a job running on _v1.64_ and the code does no longer compile:
```bash
error[E0793]: reference to packed field is unaligned
  --> C:\GitLab-Runner\builds\biomedit\sett-rs\.cargo\registry\src\index.crates.io-6f17d22bba15001f\win-crypto-ng-0.4.0\src\helpers\blob.rs:38:18
   |
38 |         unsafe { &self.0 }
   |                  ^^^^^^^
   |
   = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
   = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
For more information about this error, try `rustc --explain E0793`.
error: could not compile `win-crypto-ng` due to previous error
```
@emgre any chance to get this fixed soon? I mean, obviously there is even a `MR` ready which should solve the issue. What are the alternatives here?

**Comment by ribeaud** [comment]
Great! 🎉 This means we will sooner have a new release?

**Comment by emgre** [comment]
Yes, I'm working on that. I just want to close the currently opened PRs.

**Comment by ribeaud** [comment]
🙏

**Comment by nwalfield** [comment]
Thanks for picking this up again!

**Comment by ribeaud** [comment]
Good morning. Sorry to insist but when do you think we will have a newer release?

**Comment by emgre** [comment]
@ribeaud I just released v0.5.0
